Snippets/Core/Core_6/Errors/SecondLevel/DisableWithCode.cs

```diff
@@ -1,14 +1,13 @@
 ﻿namespace Core6.Errors.SecondLevel
 {
     using NServiceBus;
-    using NServiceBus.Features;
 
     class DisableWithCode
     {
         DisableWithCode(EndpointConfiguration endpointConfiguration)
         {
             #region DisableSlrWithCode
-            endpointConfiguration.DisableFeature<SecondLevelRetries>();
+            endpointConfiguration.SecondLevelRetries().Disable();
```

Comment:
@indualagarsamy I raised a PR to change this API here: https://github.com/Particular/NServiceBus/pull/3828 please let me know what you think about my proposal